### DISCUSSION TITLE
Remove "Upgrade to Premuim" message

### DISCUSSION THREAD

**Discussion by bladehorton** [ReVanced/revanced-discussions#182] [Question] [answered]
Very annoying when it's asks for me to upgrade to premuim when I'm using Vanced

**Comment by DraconicNEO** [comment]
Also would be good to remove the ones for Background playback in premium when background playback is disabled in settings.

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
This can now be done, but the hide function only works for older devices (pre android 10)
